### PR TITLE
Add Rawls submission option for memoryRetryMultiplier [BW-281]

### PR DESCRIPTION
**Pull request by cjllanwarne**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by cjllanwarne** [comment]
jenkins retest

File: src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala [hunk 1]
```diff
@@ -63,6 +63,7 @@ object MockWorkspaceServer {
     useCallCache = Option(randomBoolean()),
     deleteIntermediateOutputFiles = Option(randomBoolean()),
     useReferenceDisks = Option(randomBoolean()),
+    memoryRetryMultiplier = Option(randomDouble()),
```
Comment (aednichols) [inline src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala:66]: Random seems unnecessary? Could throw off someone who chooses to assert against this response in the future.

**Review by aednichols — APPROVED** [review]
👍 so long as Spray-JSON makes this change backwards compatible, i.e. clients omitting `memoryRetryMultiplier` don't cause a crash

**Comment by cjllanwarne** [comment]
jenkins retest

**Comment by cjllanwarne** [comment]
Closing, going with the simpler #857 instead

**Pull request closed without merging**